templates/accounts/describe_sounds.html

```diff
@@ -279,7 +289,7 @@ <h3>Geotag</h3>
             </p>
             <div class="gmap-wrapper" id="gmap-wrapper-{{forloop.counter}}">
                 <p>Drag this map to set a geotag:</p>
-                <div class="describe-gmap" id="gmap{{forloop.counter}}" style="width: 400px; height:300px;"></div>
+                <div class="describe-gmap geotag_edit_map_size" id="gmap{{forloop.counter}}"></div>
```

Comment:
I wouldn't make a class that specifically exists to mark the _size_ of the map. Why can't you add the size info to the `describe-gmap` class?

Related - we use `gmap` a bit here, is this geo-map or google-map?

Reply:
The use of `geotag_edit_map_size` is because the same class is used in another place where a similar map is used which does not have `describe-gmap` class. I could probably investigate and refactor to have a common class for both cases, but this code is quite old and considering that we'll refactor all once implementing maps for the new interface I did not consider refactor was worth the time. The whole maps CSS thing might in fact need some refactor. Starting by class names in which `g` stands for Google and we're no longer using Google Maps.